lib/pengine/tests/utils/Makefile.am

```diff
@@ -10,9 +10,12 @@
 include $(top_srcdir)/mk/tap.mk
 include $(top_srcdir)/mk/unittest.mk
 
+AM_CPPFLAGS += -I$(top_builddir)/lib/pengine
```

Comment:
Should we include this at a higher level than `tests/utils` to avoid duplication later, or leave it for now since only `utils` needs it currently?